guava/src/test/java/org/jdbi/v3/guava/TestGuavaCollectors.java

```diff
@@ -56,12 +58,10 @@
 
     @Before
     public void addData() {
-        ImmutableList.Builder<Integer> expected = ImmutableList.builder();
-        for (int i = 0; i < 10; i++) {
-            dbRule.getSharedHandle().execute("insert into something(name, intValue) values (?, ?)", Integer.toString(i), i);
-            expected.add(i);
-        }
-        this.expected = expected.build();
+        expected = IntStream.range(0, 10)
+            .peek(i -> dbRule.getSharedHandle().execute("insert into something(name, intValue) values (?, ?)", Integer.toString(i), i))
```

Comment:
Strange use of `peek` here

Reply:
How so? `peek` is for performing an action on each value in the stream without altering the stream content or terminating it, like logging... or running a query.